plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -90,3 +92,49 @@ def string_to_unicode(data):
         except UnicodeError:
             # try others
             continue
+
+
+def package_headers(filename):
+    """
+    Return package header.
+
+    :param filename: full path to the package to analyze
```

Comment:
It would be helpful to list which package types are supported among rpm, srpm, and drpm.